cms/templates/maintenance/_force_publish_course.html

```diff
@@ -0,0 +1,81 @@
+<%page expression_filter="h"/>
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import Text, HTML
+%>
+<div class="wrap-instructor-info studio-view command-form">
+    <form class="form-create" method="post">
+        <input type="hidden" name="csrfmiddlewaretoken" value="${csrf_token}"/>
+        <div class="wrapper-form">
+            <fieldset>
+                <ol class="list-input">
+                    <li class="field text required" id="field-course-name">
+                        <label for="course-id">${_("Course ID")}</label>
+                        <input id="course-id" type="text" name="course-id" aria-required="true"
+                               value="${form_data['course_id']}"
+                               placeholder="${_('e.g. edX+DemoX+Demo_Course')}"/>
+                        % if error:
+                        <div class="error">${msg}</div>
+                        % endif
+                    </li>
+                    <li class="field-radio">
+                        <div>
+                            <label for="dry-run" class="label" data-tooltip="${_('Trial or practice run the task')}">${_("Dry Run")}</label>
+                            % if form_data['is_dry_run']:
+                            <% is_checked = "checked" %>
+                            % else :
+                            <% is_checked = "" %>
+                            % endif
+                            <input type="checkbox" id="dry-run" name="dry-run" ${is_checked}/>
+                        </div>
+                    </li>
+                </ol>
+            </fieldset>
+        </div>
+        <div class="actions">
+            <button type="submit" class="action action-primary" aria-disabled="true">${_("Force Publish Course")}
+            </button>
+            <a href="${command['url']}" class="action action-secondary action-cancel"
+               data-tooltip="${_('Reset values')}">${_("Reset")}</a>
+        </div>
+    </form>
+    % if current_versions:
+    <div class="result">
+        % if form_data['is_dry_run']:
+        <article>
+            <div class="dry-run">
+                ${Text(_("You have {strong_start}dry run{strong_end} the course. Following course versions would be changed :")).format(
+                    strong_start=HTML('<strong>'),
+                    strong_end=HTML('</strong>'),
+                )}
+            </div>
+            <div class="main-output">
+                ${_("Published branch version {published} changed to draft branch version {draft}").format(
```

Comment:
`The published branch version, {published}, was reset to the draft branch version, {draft}`